evals/solvers/providers/google/gemini_solver_wav.py

```diff
@@ -0,0 +1,164 @@
+import base64
+import copy
+import os
+from dataclasses import asdict, dataclass
+import re
+from typing import Any, Dict, Union
+
+import google.api_core.exceptions
+import google.generativeai as genai
+from google.generativeai.client import get_default_generative_client
+
+import numpy as np
+
+from evals.solvers.solver import Solver, SolverResult
+from evals.task_state import Message, TaskState
+from evals.utils.api_utils import create_retrying
+
+from evals.solvers.providers.google.gemini_solver import SAFETY_SETTINGS, GEMINI_RETRY_EXCEPTIONS, GeminiSolver
+
+# Load API key from environment variable
+API_KEY = os.environ.get("GEMINI_API_KEY")
+genai.configure(api_key=API_KEY)
+
+def _data_url_to_wav(url):
+    if not url.startswith("data:"):
+        raise ValueError("Not a data URL")
+    _, raw_data = url.split(",", 1)
+    return base64.b64decode(raw_data)
```

Comment:
nit: I see this in `evals/solvers/providers/openai/realtime_solver.py` as well.
I'd suggest reusing the same function (feel free to refactor it to a shared util file).

Reply:
Moved to utils.py